app/src/main/res/xml/keyboard_qwerty_german.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Keyboard xmlns:android="http://schemas.android.com/apk/res/android"
+    android:horizontalGap="3dp"
+    android:verticalGap="6dp"
+    android:keyWidth="34dp"
+    android:keyHeight="34dp">
+    <Row>
+        <Key android:codes="113" android:keyLabel="q" android:keyEdgeFlags="left" android:popupKeyboard="@xml/keyboard_popup" android:popupCharacters="ʠɋq̃"  />
+        <Key android:codes="119" android:keyLabel="w" android:popupCharacters="wẃẁŵẅ" android:popupKeyboard="@xml/keyboard_popup" />
+        <Key android:codes="101" android:keyLabel="e" android:popupCharacters="eĕḝȇêéę" android:popupKeyboard="@xml/keyboard_popup" />
```

Comment:
I'll let a German speaker correct me, but I believe `€` should be a popup modifier character for `e`